euth/communitydebate/templates/euth_communitydebate/topic_form.html

```diff
@@ -55,6 +55,18 @@ <h3 class="sans-serif">
                 {% endif %}
                 {{ form.description.errors }}
             </div>
+
+            <div class="form-group">
+                <div class="input-group form-control-upload row">
+                    <div class="col-sm-8 col-md-9">
+                        <label>Upload a picture</label>
+                        <p class="text-warning">Please make sure the image is at least 800x200 px in dimensions and at most 5 MB in file size.</p>
```

Comment:
Even though it's the same in the idea form, I think this is not the proper way to do it. No need to fix that in this PR, though. It's not even translated? Weird!